jira.yml

```diff
@@ -0,0 +1,6 @@
+
+- hosts: all 
+  tasks:
+    - shell: curl -IL tut.by
```

Comment:
* [**E101**] Instead of always_run, use check_mode.

* [**E103**] Instead of sudo/sudo_user, use become/become_user.

* [**E301**] Commands should either read information (and thus set changed_when) or not do something if it has already been done (using creates/removes) or only do it if another check has a particular result (when)

* [**E303**] Executing a command when there is an Ansible module is generally a bad idea

* [**E305**] Shell should only be used when piping, redirecting or chaining commands (and Ansible would be preferred for some of those!)

* [**E502**] All tasks should have a distinct name for readability and for --start-at-task to work